Java/Passport/Passport.java

```diff
@@ -113,29 +118,29 @@ public String getSurname() {
     }
 
     public Date getbDate() {
-        return bDate;
+        return birthDate;
     }
 
-    private static void validateSeries(char[] buffer)
+    private static void validateSeries(char[] series)
             throws InvalidSerialException {
-        for ( int i = 0; i < SIZE; i++ ) {
-            if ( Character.isLowerCase(buffer[i]) ) {
-                buffer[i] = Character.toUpperCase(buffer[i]);
+        for ( int i = 0; i < SERIES_LENGTH; i++ ) {
+            if ( Character.isLowerCase(series[i]) ) {
+                series[i] = Character.toUpperCase(series[i]);
```

Comment:
Probably `Character.isLowerCase(series[i])` is unnecessary. I guess if char is already upper, then `Character.toUpperCase(series[i])` will just ignore that